src/components/Menu.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import '../styles/menu.css'
+
+class Menu extends Component {
+  render() {
+    if (this.props.active) {
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```